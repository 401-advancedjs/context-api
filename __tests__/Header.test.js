import React from 'react';
import Header from '../component/Header/Header.js';

import renderer from 'react-test-renderer';

describe('<Header />', () => {
  test('basic rendering', () => {
    const mountedHeader = shallow(<Header />);
    expect(mountedHeader.find('Header')).toBeTruthy();
  });

  test('rendering follows the snapshot', () => {
    const snapshot = renderer.create(<Header />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
