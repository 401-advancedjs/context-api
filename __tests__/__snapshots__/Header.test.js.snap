// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> rendering follows the snapshot 1`] = `
<header>
  <h1>
    RESTy
  </h1>
</header>
`;
